wiki/Reporting_Bad_Behaviour/en.md

```diff
@@ -1,133 +1,112 @@
 
 Remember to add details to your report (like translations) at the comments.
 
-<center>
-<img src="Report-user-profile.png" title="Report-user-profile.png" alt="Report-user-profile.png" width="400" />
+![This will open a dialog for reporting the user.](img/Report-user-profile.png "This will open a dialog for reporting the user.")
 
-</center>
-### Using the report link at the user's record at beatmap listing
 
-Only use this when you are reporting users for cheating.
+### Using the report link at the record of a user at beatmap listing
 
-<center>
-<img src="Report-user-beatmap.png" title="Report-user-beatmap.png" alt="Report-user-beatmap.png" width="400" />
+This should only be used when cheating users are reported.
+
+![Pressing this link will report that score.](img/Report-user-beatmap.png "Pressing this link will report that score.")
 
-</center>
 
-------------------------------------------------------------------------
 
 Examples what to report
 -----------------------
 
-:\* **Spamming or flooding in chat.**
-
-Using a broken keyboard is not an excuse!
+* **Spamming or flooding in chat.**
 
-:\* **Excessive use of uppercase letters.**
+ Using a broken keyboard is not an excuse!
 
-This has been considered as shouting and annoying.
+* **Excessive use of uppercase letters.**
 
-:\* **Posting of inappropriate contents such as pornographic materials (questionable/suggestive are considerable).**
+ This has been considered as shouting and annoying.
 
-Moderators immediately silence for this, removing the content without warning.''
+* **Posting of inappropriate contents such as pornographic materials (questionable/suggestive are considerable).**
 
-:\* **Posting of unwanted contents such as advertisements, screamers and referrals.**
+ Moderators immediately silence for this, removing the content without warning.
 
-Take note that spamming streaming links are applied to this rule.
+* **Posting of unwanted contents such as advertisements, screamers and referrals.**
 
-:\* **Offensive behavior towards other players.**
+ Take note that spamming streaming links are applied to this rule.
 
-Telling people to kill themselves is not a good thing to say.
+* **Offensive behavior towards other players.**
 
-<!-- -->
+ Telling people to kill themselves is not a good thing to say.
 
 **and many more..**
 
-------------------------------------------------------------------------
 
 Frequently Asked Questions
 --------------------------
 
-**Can other players see my reports in chat?**
-
-Obviously, no, unless you typed the command prefix wrong. Make sure you type an exclamation mark first!
+* **Can other players see my reports in chat?**
 
-<!-- -->
+ Obviously, no, unless you typed the command prefix wrong. Make sure you type an exclamation mark first!
 
-**I accidentally reported a user! Will I get an infringement?**
 
-No. Don't worry about it, we can ignore false reports unless abused.
+* **I accidentally reported a user! Will I get an infringement?**
```

Comment:
Infringement means the active breaking of laws - not the best word to use here. Perhaps "punishment" works better.